connectors/ssh/src/main/java/org/crsh/ssh/term/CRaSHCommandFactory.java

```diff
@@ -40,4 +40,9 @@ public CRaSHCommandFactory(ShellFactory shellFactory, Charset encoding) {
   public Command create() {
     return new CRaSHCommand(this);
   }
+
+  @Override
+  public Command get() {
+    return new CRaSHCommand(this);
```

Comment:
Why is this new `get()` needed vs `create()`. I don't know much about it, just asking. Could we delegate to `create()` in case?